jobs/build-charms/builder_local.py

```diff
@@ -550,12 +551,39 @@ def pull_layers(self):
 @unique
 class Arch(Enum):
     ALL = "all"
+    UNKNOWN = "unknown"
     AMD64 = "amd64"
     ARM64 = "arm64"
     ARMHF = "armhf"
     PPC64EL = "ppc64el"
     S390X = "s390x"
 
+    @classmethod
+    def from_value(cls, value: str) -> "Arch":
+        for arch in cls:
+            if arch.value == value:
+                return arch
+        if value == "arm":
+            return arch.ARMHF
+        return Arch.UNKNOWN
+
+
+@unique
+class Series(Enum):
+    ALL = "all"
+    UNKNONWN = "unknown"
```

Comment:
```suggestion
    UNKNOWN = "unknown"
```
Maybe a tiny typo here?